### PR TITLE
fix tests

### PR DESCRIPTION
**Pull request by simar7**

### CODE REVIEW

**Comment by simar7** [comment]
@nikpivkin could you take a look at reviewing this?

File: test/docker_test.go [hunk 1]
```diff
@@ -4,68 +4,104 @@ import (
...
+			return nil
+		})
+
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func Test_Docker_RegoPoliciesFromDisk(t *testing.T) {
```
Comment (nikpivkin) [inline test/docker_test.go:54]: This test loads Rego rules from the file system, but we moved the rules to another repo, so do we need this test? There are other tests that use custom rules.

Comment (simar7) [reply test/docker_test.go]: OK for the moment I've commented out until we figure out what we can (if anything) do with it. https://github.com/aquasecurity/trivy-iac/pull/6/commits/c8beec6a587775151ee1a564fa34dd9c0a0fc76d

**Comment by nikpivkin** [comment]
@simar7 I restarted the tests and they completed successfully. The problem was related to the terraform module cache (reproduced locally). We can add a new option for terraform parser that will disallow using the cache to load modules.

**Review by nikpivkin — APPROVED** [review]
LGTM

**Pull request merged**